README.md

```diff
@@ -1,37 +1,58 @@
+* Both messages and emojis appear over each users' heads.
+* Emojis can be spammed to your hearts content!
+* Toggle a switch to stop sharing your location in Incognito Mode.
+* See how many users are connected using PubNub's Presence Badge!
 
-"Image of App"
+![pubmoji gif](https://pubnubdevelopers.github.io/PubMoji-Product-Page/img/demo.gif)
 
-All user-data is handled securley and reliably through PubNub. Each user publishes their data across a PubNub channel of which the other users receive in realtime by subscribing to that channel. This entails instances such as rendering people on a map based on the GPS coordinates they Published or tracking each user's state by having them publish user-specific values such as UUID, GPS permission, etc. 
+All user data is handled [securely](https://www.pubnub.com/developers/tech/security/) and [reliably](https://www.pubnub.com/blog/build-a-reliable-product-that-never-fails/) through PubNub. Each user publishes their data across a PubNub channel, and other users who are subscribed to that channel receive those messages in realtime. 
 
 # How to run
 
- ```npm i```
+- [ ] Make sure that you have React Native set up in your machine. If you don't, then follow the [React Native CLI Quickstart](https://facebook.github.io/react-native/docs/getting-started.html) instructions in the official documentation.
 
- ```react-native link```
- 
- ```react-native run-ios``` or ```react-native run-android```
+- [ ] Clone this repository to your computer:
+ ```bash
+    git clone https://github.com/PubNubDevelopers/PubMoji.git
+  ```
 
-"Image of PubNub Abstraction"
-      
-      
-#### To get started, sign up for a free PubNub account to get your Pub/Sub keys!
+- [ ] Replace the placeholder keys on line 37 and 38 of your App.js with your own PubNub keys through the link below!
 
-"PubNub Key image link"
+<a href="https://dashboard.pubnub.com/signup?devrel_gh=PubMoji">
+    <img alt="PubNub Signup" src="https://i.imgur.com/og5DDjf.png" width=260 height=97/>
+</a>
 
+- [ ] These next steps need to be completed if you would like to run PubMoji on Android
+    - [ ] [Get a Google Maps API](https://developers.google.com/maps/documentation/javascript/get-api-key) 
+    - [ ] Create the file 'gradle.properties' and 'local.properties' under the 'android' folder. Insert the following lines into them:  
+    - [ ] ```Google_Maps_ApiKey="YOUR_API_KEY"``` into 'gradle.properties', replacing the placeholder with your Google Maps API key. 
+    - [ ] ```sdk.dir = /Users/USERNAME/Library/Android/sdk``` into 'local.properties', replacing USERNAME with your computer username. Check out [this stackoverflow question](https://stackoverflow.com/questions/32634352/react-native-android-build-failed-sdk-location-not-found) for help with other operating systems besides MacOS. 
+
+- [ ] Next install all the required packages using ```npm i```
+
+- [ ] Link packages to your project with ```react-native link```
```

Comment:
maybe a disclaimer about some packages may need to be manually linked if not linked automatically with react-native link